### PR TITLE
Add a chart for Mercure.rocks

### PR DESCRIPTION
**Pull request by dunglas**
#### What this PR does / why we need it:

This PR adds a chart for [the Mercure hub](https://mercure.rocks).

Mercure is a protocol allowing to push data updates to web browsers and other HTTP clients in a convenient, fast, reliable and battery-efficient way. It is especially useful to publish real-time updates of resources served through web APIs, to reactive web and mobile apps.

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mercure/templates/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  annotations:
+    helm.sh/hook: "pre-install"
+  labels:
+    app.kubernetes.io/name: {{ include "chart.name" . }}
+    helm.sh/chart: {{ include "chart.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+type: Opaque
+data:
+  jwtKey: {{ if .Values.jwtKey }}{{ .Values.jwtKey | b64enc | quote }}{{ else }}{{ randAlphaNum 12 | b64enc | quote }}{{ end }}
```
Comment (vincentchalamon) [inline stable/mercure/templates/secret.yaml]:
```suggestion
  jwtKey: {{ .Values.jwtKey | default (randAlphaNum 12) | b64enc | quote }}
```

**Review by vincentchalamon — CHANGES_REQUESTED** [review]

**Comment by dunglas** [comment]
Hi there, this chart is ready to be merged. I tested it several time using different options and it works.

**Review by clementtalleu — APPROVED** [review]

File: stable/mercure/OWNERS [hunk 2]
```diff
@@ -0,0 +1,2 @@
+approvers:
+- dunglas
```
Comment (cpanato) [inline stable/mercure/OWNERS:2]: add the reviewers as well

Comment (dunglas) [reply stable/mercure/OWNERS:2]: You mean @vincentchalamon and @clementtalleu? Or people from the Helm team?

File: stable/mercure/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "mercure.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "mercure.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
```
Comment (cpanato) [inline stable/mercure/templates/deployment.yaml:21]: add `helm.sh/chart` / `app.kubernetes.io/managed-by`

**Review by cpanato — CHANGES_REQUESTED** [review]
see small comments above

File: stable/mercure/OWNERS (hunk 2 above)
Comment (cpanato) [reply stable/mercure/OWNERS:2]: can be you as well since you are the approver

**Comment by dunglas** [comment]
Thank you very much @cpanato! All comments addressed.

File: stable/mercure/values.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+demo: "0"
+jwtKey: ""
+logFormat: "FLUENTD"
+publishAllowedOrigins: ""
+publisherJwtKey: ""
+subscriberJwtKey: ""
+
+image:
+  repository: dunglas/mercure
+  tag: latest
```
Comment (cpanato) [inline stable/mercure/values.yaml]: please do not use `latest` lets use a stable version

Comment (dunglas) [reply stable/mercure/values.yaml]: done

File: stable/mercure/.helmignore [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
```
Comment (cpanato) [inline stable/mercure/.helmignore:22]: add the OWNERS here please

File: stable/mercure/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: "0.3.3"
+description: The Mercure hub allows to push data updates using the Mercure protocol to web browsers and other HTTP clients in a convenient, fast, reliable and battery-efficient way
+name: mercure
+version: 0.1.0
```
Comment (cpanato) [inline stable/mercure/Chart.yaml]: lets set to 1.0.0 since it is in stable
```suggestion
version: 1.0.0
```

**Review by cpanato — CHANGES_REQUESTED** [review]
just smalls adjustments
thanks for your work on this

File: stable/mercure/.helmignore (hunk 5 above)
Comment (dunglas) [reply stable/mercure/.helmignore:22]: Should be all set now :)

**Review by cpanato — APPROVED** [review]

**Comment by dunglas** [comment]
@cpanato the icon isn't rendered on the Helm website: https://hub.helm.sh/charts/stable/mercure
Do you know if it's because it's a SVG file or if it's because of its format (or both)? Thanks!

**Pull request merged**